### ISSUE TITLE
Design Bugs

### ISSUE THREAD

**Issue by WebSavvyGuy** [RocketChat/Rocket.Chat#7733]
### Description:

There are all sorts of design bugs.   For example:  Userlist is not showing up on certain browser sizes.  Full width in browsers are ok (userlist shows) but as you move the browser window smaller the userlist doesn't show anymore.   Another issue is some padding problems on the "Register or Chat Anonymously" buttons.  They sit too low on the browser.  Another issue is the box when you select a username has some margin issues.   

### Server Setup Information:

* Version of Rocket.Chat Server: 0.58.rc.1
* Operating System:
* Deployment Method(snap/docker/tar/etc):
* Number of Running Instances: 
* DB Replicaset Oplog:
* Node Version:

### Steps to Reproduce:

Simply test it on any browser or mobile device.

### Expected behavior:

No design bugs.

### Actual behavior:

Many design bugs.

**Comment by jszaszvari** [comment]
| as you move the browser window smaller the userlist doesn't show anymore

I'm pretty sure that's by design, it switches to the mobile interface when the size of your browser reaches a certain size as to still be able to chat to people and you can press the hamburger menu that appears in the top left to bring the sidebar back. Although I'm just guessing that what is happening without screenshots 

There is a huge redesign underway at the moment that should fix a lot of this 

See here https://github.com/RocketChat/Rocket.Chat/pull/7643/

**Comment by jszaszvari** [comment]
Until that's done,  you can fix some of the Ui problems by using the Override CSS functionality in the admin sexton

**Comment by WebSavvyGuy** [comment]
@jszaszvari    What do you mean the override CSS functionality?

Also, as the browser window goes smaller it should still be able to show the userlist if you click it but it simply doesn't appear at all in mobile devices.

**Comment by jszaszvari** [comment]
See gifs below, Is this what you are having a issue with? 

Mobile:
![c](https://user-images.githubusercontent.com/872463/29251607-fb524694-809a-11e7-9528-a728cf9a65be.gif)

![size](https://user-images.githubusercontent.com/872463/29251623-5c300302-809b-11e7-9e70-38d218d80184.gif)

By custom css, I was talking about how you can modify the CSS application wide in the settings like this - So if a button is misaligned, until the redesign is finished you can most likely fix it with CSS: 
![2017-08-14_02-27-43-1](https://user-images.githubusercontent.com/872463/29251631-a697491e-809b-11e7-9a57-4db99ce4e349.gif)

**Comment by WebSavvyGuy** [comment]
@jszaszvari Thanks alot for your nice gifs.   I need to figure out how to do that to show you guys.

But my issue is when i press the "userlist" icon (the two heads) normally in mobile or smaller browsers it should still open up the list so i can actually see whos in the room.  However, it is no longer doing that.

**Comment by pandada8** [comment]
![image](https://user-images.githubusercontent.com/3305174/29263921-97ee16dc-810d-11e7-8cdf-8a74e539ce48.png)

this is an example from official demo, where I clicked the button in sidebar and nothing showed,
adding `height: 100%` to `@media (max-width: 1100px) .flex-tab-container.opened .flex-tab` can sort of fix the problem.

![image](https://user-images.githubusercontent.com/3305174/29264021-0254561c-810e-11e7-8042-9d87e9cd1867.png)

**Comment by WebSavvyGuy** [comment]
@pandada8 

Do you mind to share where you made those changes please

**Comment by WebSavvyGuy** [comment]
I added the following into Layout>Custom CSS

`@media(max-width:1100px) {
 .flex-tab-container.opened .flex-tab {
  position:absolute;
   right:40px;
   border-width:0 0 0 1px;
   z-index:100;
   height:90%
  }
}`

Because without seeing the userlist, nobody was able to PM anyone else.
It fixed the user list issue in mobile but now people are complaining the avatars are so big and taking the whole mobile screen or something and they cant type.    I noticed these issues on both IPhone and Samsung.
